frontend-react/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from "@trussworks/react-uswds";
+import classnames from "classnames";
+
+// Can't seem to import these directly from react-uswds? Copied here instead.
+export interface ButtonProps {
+    type: "button" | "submit" | "reset";
+    children: React.ReactNode;
+    secondary?: boolean;
+    base?: boolean;
+    /**
+     * @deprecated since 1.15.0, use accentStyle
+     */
+    accent?: boolean;
+    accentStyle?: "cool" | "warm";
+    outline?: boolean;
+    inverse?: boolean;
+    size?: "big" | "small"; // small is deprecated
+    /**
+     * @deprecated since 1.6.0, use size
+     */
+    big?: boolean;
+    /**
+     * @deprecated since 1.6.0, use size
+     */
+    small?: boolean;
+    /**
+     * @deprecated since 1.9.0
+     */
+    icon?: boolean;
+    unstyled?: boolean;
+}
+
+export interface IconButtonProps extends ButtonProps {}
```

Comment:
Do we need this?

Reply:
I feel it's good practice for every component to have a props interface exposed, even if in this case there's nothing added (yet) on top of ButtonProps. However, I can drop if desired.

Reply:
Sure, not a problem in my mind!